lib/widgets/screens/home/users_list.dart

```diff
@@ -98,14 +100,17 @@ class UsersList extends StatelessWidget {
         return Text('Error: $error');
       },
       onData: (data, {refetch, fetchMore}) {
-        List users = data.response != null
+        List<Query$FindAllUsers$findUsers> users = data.response != null
             ? data.response!.reversed
                 .where((element) => element.id != currentUser?.id)
                 .toList()
             : [];
         users = users
             .where((element) =>
-                element.fullName.toLowerCase().contains(search.toLowerCase()))
+                element.fullName
```

Comment:
We'll need to provide the query to do a paginated search (limit this to X results)

Reply:
Yes definitely! Here I was just refactoring the existing code to use the generated models. Pagination should be done separately.

Reply:
Yes, this is something Holger is working on today in the backend